src/effects/Tween.js

```diff
@@ -58,9 +58,7 @@ Tween.propHooks = {
 	_default: {
 		get: function( tween ) {
 			var result;
-
-			if ( tween.elem[ tween.prop ] != null &&
-				(!tween.elem.style || tween.elem.style[ tween.prop ] == null) ) {
+			if ( !tween.elem.style || tween.elem[ tween.prop ] != null && tween.elem.style[ tween.prop ] == null) {
```

Comment:
We've settled on `nodeType` as the DOM node discriminator... I'd say that means replacing `!tween.elem.style` with `tween.elem.nodeType !== 1` here, and `tween.elem.style` with `tween.elem.nodeType === 1` in `set`. There's also some rearranging to make them more compressible, but that can come after the logic update.